traffic_ops/traffic_ops_golang/riak.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/tostructs"
+	"github.com/basho/riak-go-client"
+	"github.com/jmoiron/sqlx"
+	"github.com/lestrrat/go-jwx/jwk"
+	"io/ioutil"
+	"net/http"
+)
+
+const riak_port = 8087
+const cdn_uri_keys_bucket = "cdn_uri_sig_keys" // riak namespace for cdn uri signing keys.
+
+func getStringValue(resp *riak.FetchValueResponse) (string, error) {
```

Comment:
`getStringValue` is maybe a bit generic of a name for a global function that converts a specific library type into a  string. It's only used in one spot as near as I can tell; you might put it in local scope of the function that uses it.

Reply:
It may be used with other objects such as the url_sig keys going forward.  Thats why its there.